src/main/java/net/sf/jabref/importer/fileformat/BiblioscapeImporter.java

```diff
@@ -41,12 +41,16 @@ public String getFormatName() {
 
     @Override
     public List<String> getExtensions() {
-        return null;
+        List<String> extensions = new ArrayList<>();
```

Comment:
extract the list as a field. Otherwise, everytime when the method is called, a list is initalized.